src/core/aws-ses.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import * as AWS from 'aws-sdk';
+import { EnvironmentService } from './config/environment.service';
+
+@Injectable()
+export class SesService {
+  private readonly ses: any;
+  constructor(private readonly env: EnvironmentService) {
+    AWS.config.update({
+      accessKeyId: this.env.string('AWS_ACCESS_KEY').optional('cord-field'),
+      secretAccessKey: this.env.string('AWS_SECRET_KEY').optional('cord-field'),
+      region: this.env.string('AWS_REGION').optional('cord-field'),
+    });
```

Comment:
We don't want to be changing a global singleton in this specific service (`update()`).

Also, _access key id_ and _secret access key_ will automatically be pulled from the environment or a number of different places automatically. So we don't want to limit ourselves to only env vars.

I believe the default region is also determined in a similar way, but if not that should be moved to `ConfigService` just like `emailFrom` and defaulted to `us-east-2`. That still can be passed directly into the `SES` constructor instead of the global `update()` call.